strings/build.gradle

```diff
@@ -34,7 +33,11 @@ android {
 
 dependencies {
     implementation Dependencies.androidx_core_ktx
-    implementation Dependencies.androidx_appcompat
+
+    /**
+     * We need to expose [AppCompatActivity] for classes that extend [LocalizedActivity]
```

Comment:
I would edit it a bit:

We need to expose [AppCompatActivity] for classes **in separate modules** that extend [LocalizedActivity]